backoffice/src/components/backoffice/organisation/OrganisationPanel.jsx

```diff
@@ -408,7 +418,7 @@ export default class OrganisationPanel extends React.Component {
                                                 }
                                                 {(advice.reponse && !this.state.reply.shown) &&
                                                 <div className="answer">
-                                                    <h4>Votre réponse</h4>
+                                                    <h4>Votre réponse <span>({this.getStatus(advice.reponse)})</span></h4>
                                                     <p>{advice.reponse.text}</p>
```

Comment:
du coup quand c'est en attente de modération, on affiche aussi le text ?

Reply:
oui on l'affiche sur le BO Financeur et OF. La réponse est visible par tous dans l'export.